### PR TITLE
New device for existing brands: Vivo, Huawei, Oppo

### PR DESCRIPTION
**Pull request by Wujiao233**

### CODE REVIEW

File: Tests/fixtures/smartphone-22.yml [hunk 1]
```diff
@@ -7278,3 +7278,399 @@
...
+  client:
+    type: browser
+    name: Chrome Webview
+    version: 77.0.3865.120
+    engine: Blink
+    engine_version: ""
+  device:
+    type: smartphone
+    brand: Huawei
+    model: Honor V40
```
Comment (sanchezzzhak) [inline Tests/fixtures/smartphone-22.yml:7404]: set model `Honor V40 Lite`

File: Tests/fixtures/smartphone-22.yml [hunk 2]
```diff
@@ -7278,3 +7278,399 @@
...
+  client:
+    type: browser
+    name: Chrome Webview
+    version: 77.0.3865.120
+    engine: Blink
+    engine_version: ""
+  device:
+    type: smartphone
+    brand: Huawei
+    model: Honor 30
```
Comment (sanchezzzhak) [inline Tests/fixtures/smartphone-22.yml:7674]: Honor 30 Lite

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

File: Tests/fixtures/smartphone-22.yml (hunk 2 above)
Comment (Wujiao233) [reply Tests/fixtures/smartphone-22.yml]: Done.

File: regexes/device/mobiles.yml [hunk 3]
```diff
@@ -6790,6 +6800,8 @@ Huawei:
       model: 'Honor 30S'
     - regex: 'LRA-LX1(?:[);/ ]|$)'
       model: 'Honor 30I'
+    - regex: 'MXW-[AT]N00(?:[);/ ]|$)'
+      model: 'Honor 30 Lite'
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:6804]: remove the space char after " Honor 30 Lite"

**Review by sanchezzzhak — CHANGES_REQUESTED** [review]

**Review by sanchezzzhak — APPROVED** [review]

**Comment by sanchezzzhak** [comment]
thank you for your contribution

**Pull request merged**